.github/workflows/load-test.yml

```diff
@@ -0,0 +1,76 @@
+name: Load Test
+
+on:
+  pull_request:
+    branches:
+      - develop
+  push:
+    branches:
+      - develop
+    tags:
+      - v*
+  workflow_dispatch:
+
+env:
+  AERIE_USERNAME: "aerie"
+  AERIE_PASSWORD: "aerie"
+  HASURA_GRAPHQL_ADMIN_SECRET: "aerie"
+  HASURA_GRAPHQL_JWT_SECRET: '{ "type": "HS256", "key": "oursupersecretsupersecurekey1234567890" }'
+  POSTGRES_USER: "postgres"
+  POSTGRES_PASSWORD: "postgres"
```

Comment:
To avoid collision of test runs / results, load testing was moved to a new namespaced environment. This required redefining environment variables here or creating new secrets in the repo settings, since env vars are separated per environment in GH Actions.

Reply:
> To avoid collision of test runs / results, load testing was moved to a new namespaced environment.

Can you clarify this? I'm not very familiar with GH environments.

Reply:
Our e2e-test results currently show up under the [`e2e-test` deployment environment results](https://github.com/NASA-AMMOS/aerie/deployments/e2e-test), since we use the `e2e-test` environment in `test.yml`. I figured namespacing the load tests into a new environment would make sense to avoid the confusion of different test results showing up in the same section.

Unfortunately, the secret env vars we defined for `e2e-test` are scoped to that specific GH environment, so I had to redefine them here.